app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -186,7 +189,16 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         components.publicSuffixList.prefetch()
 
         setupThemeAndBrowsingMode(getModeFromIntentOrLastKnown(intent))
-        setContentView(R.layout.activity_home)
+        setContentView(R.layout.activity_home).run {
+            //The NavGraph is being asynchronously inflated for performance gain.
+            //However, it has to be called immediately after the setContentView since it requires
```

Comment:
Can this, "it needs to be called because of thesve reasons" go into the kdoc for the class? The reasons aren't really appropriate for inline code. WHat we need to convey in inline code is, "This function needs to be called RIGHT HERE and SHOULD NOT BE MOVED. See kdoc for deteails"